tests/NLog.UnitTests/LayoutRenderers/RegistryTests.cs

```diff
@@ -196,6 +263,67 @@ public void RegistyDefaultValueTest_with_foward_slash()
             AssertLayoutRendererOutput("${registry:value=NOT_EXISTENT:key=HKLM/NOT_EXISTENT:defaultValue=C\\\\temp}",
                 "C\\temp");
         }
+
+
+        [Fact]
+        public void RegistyDefaultValueTest_with_foward_slash2()
```

Comment:
Many test method names (including some old ones) start with `Registy` instead of `Registry`.